pages/common/csvpy.md

```diff
@@ -0,0 +1,12 @@
+# csvpy
+
+> Loads a CSV file into a Python shell.
```

Comment:
To anyone else mystified by what this does: it drops in a shell and the prints a friendly message saying you can access your file through the `reader` variable. Self-explanatory enough once you try it out.